internal/commands/config/get_internal_test.go

```diff
@@ -0,0 +1,139 @@
+package config_internal
+
+import (
+	"slices"
+	"testing"
+
+	"github.com/spf13/viper"
+)
+
+// Test RunInternalConfigGet function
+func Test_RunInternalConfigGet_NoArgs(t *testing.T) {
+	args := []string{}
+	if err := RunInternalConfigGet(args); err != nil {
+		t.Errorf("Error running internal config get: %s", err.Error())
+	}
+}
+
+// Test RunInternalConfigGet function with args that are set
+func Test_RunInternalConfigGet_WithArgs(t *testing.T) {
+	viper.Set("pingone.worker.clientId", "test-client-id")
+
+	args := []string{"pingone.worker.clientId"}
+	if err := RunInternalConfigGet(args); err != nil {
+		t.Errorf("Error running internal config get: %s", err.Error())
+	}
+}
+
+// Test RunInternalConfigGet function with args that are not set
+func Test_RunInternalConfigGet_WithArgs_NotSet(t *testing.T) {
+	args := []string{"pingone.worker.clientId"}
+	if err := RunInternalConfigGet(args); err != nil {
+		t.Errorf("Error running internal config get: %s", err.Error())
+	}
+}
+
+// Test RunInternalConfigGet function with invalid key
+func Test_RunInternalConfigGet_InvalidKey(t *testing.T) {
+	args := []string{"pingctl.invalid"}
+	if err := RunInternalConfigGet(args); err == nil {
+		t.Errorf("Expected error running internal config get")
+	}
+}
+
+// Test RunInternalConfigGet function with too many args
+func Test_RunInternalConfigGet_TooManyArgs(t *testing.T) {
+	args := []string{"pingone.worker.clientId", "pingone.worker.clientSecret"}
+	if err := RunInternalConfigGet(args); err != nil {
+		t.Errorf("Error running internal config get: %s", err.Error())
+	}
+}
+
+// Test parseGetArgs function
+func Test_parseGetArgs(t *testing.T) {
+	args := []string{"pingone.worker.clientId"}
+	if _, err := parseGetArgs(args); err != nil {
+		t.Errorf("Error parsing get args: %s", err.Error())
+	}
+}
+
+// Test parseGetArgs function with no args
+func Test_parseGetArgs_NoArgs(t *testing.T) {
+	args := []string{}
+	if _, err := parseGetArgs(args); err != nil {
+		t.Errorf("Error parsing get args: %s", err.Error())
+	}
+}
+
+// Test parseGetArgs function with too many args
+func Test_parseGetArgs_TooManyArgs(t *testing.T) {
+	args := []string{"pingone.worker.clientId", "pingone.worker.clientSecret"}
+	if _, err := parseGetArgs(args); err != nil {
+		t.Errorf("Error parsing get args: %s", err.Error())
+	}
+}
+
+// Test PrintConfig() function
+func ExamplePrintConfig() {
+	// set viper configuration key-value for testing
+	viper.Set("pingctl.color", true)
+	viper.Set("pingctl.output", "text")
+	viper.Set("pingone.region", "test-region")
+	viper.Set("pingone.worker.clientId", "test-client-id")
+	viper.Set("pingone.worker.clientSecret", "test-client-secret")
+	viper.Set("pingone.worker.environmentId", "test-environment-id")
+	viper.Set("pingone.export.environmentId", "test-export-environment-id")
+
+	_ = PrintConfig()
+
+	// Output:
```

Comment:
Should these tests be validating this output?

Reply:
They are validating with https://pkg.go.dev/testing#hdr-Examples

Reply:
Woah that's cool, and built in to the main go testing package too. Thanks 👍